pontoon/base/forms.py

```diff
@@ -397,3 +398,32 @@ class AddCommentForm(forms.Form):
     entity = forms.IntegerField(required=False)
     comment = HtmlField()
     translation = forms.IntegerField(required=False)
+
+
+class CreateTokenForm(forms.ModelForm):
+    """
+    Form for creating Personal Access Tokens.
+    """
+
+    name = forms.RegexField(
+        regex=r"^[a-zA-Z0-9-]+$",
+        error_messages={
+            "invalid": "Name may only contain letters, numbers, and hyphens."
+        },
+    )
```

Comment:
Why do we use this limitation? Why not just allow any character the model field allows? I think GitHub does the same.